bundles/org.openhab.binding.vallox/src/main/resources/ESH-INF/thing/se-channels.xml

```diff
@@ -0,0 +1,502 @@
+		<category>Temperature</category>
+		<state pattern="%.1f %unit%" readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seFilterGuardIndicator" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Filter Guard Indicator</label>
+		<description>Indicate filter might be clogged. Check filter!</description>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="sePostHeatingIndicator" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Post Heating Indicator</label>
+		<description>Indicate that heating is active</description>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seFaultIndicator" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Fault Indicator</label>
+		<description>Indicate that fault is active</description>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seServiceReminderIndicator">
+		<item-type>Switch</item-type>
+		<label>Service Reminder Indicator</label>
+		<description>Indicate that maintenance is needed</description>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seHumidity" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity</label>
+		<description>Highest measured relative humidity.</description>
+		<state pattern="%d %unit%" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seBasicHumidityLevel" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Basic Humidity Level</label>
+		<description>When humidity adjust on ON, this is the target humidity what the machine tries to achieve</description>
+		<state min="0" max="100" pattern="%d %unit%" readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seHumiditySensor1" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity Sensor 1</label>
+		<description>Relative Humidity measured from sensor 1</description>
+		<state pattern="%d %unit%" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seHumiditySensor2" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity Sensor 2</label>
+		<description>Relative Humidity measured from sensor 2</description>
+		<state pattern="%d %unit%" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seCo2" advanced="true">
+		<item-type>Number</item-type>
+		<label>Measured Co2</label>
+		<description>Highest measured Co2 value from sensor(s)</description>
+		<state min="0" max="2000" step="1" pattern="%d ppm" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seCo2SetPoint" advanced="true">
+		<item-type>Number</item-type>
+		<label>Co2 Setpoint</label>
+		<description>Setpoint for Co2 adjustment</description>
+		<state min="0" max="2000" step="1" pattern="%d ppm" readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seAdjustmentInterval" advanced="true">
+		<item-type>Number</item-type>
+		<label>Adjustment interval</label>
```

Comment:
The first letter of every word in a label should be capitalized.

Reply:
Done.